### PR TITLE
Add UKI functionlity to replace titles

### PR DESCRIPTION
**Pull request by mauromorales**
This is the resulting boot menu titles

Screenshot 2024-03-12 at 10 50 01

relates to kairos-io/kairos#2325

### CODE REVIEW

File: pkg/uki/upgrade.go [hunk 1]
```diff
@@ -121,6 +121,15 @@ func (i *UpgradeAction) installRecovery() error {
 	if err != nil {
 		return err
 	}
+	err = replaceRoleInKey(targetConfPath, "efi", UnassignedArtifactRole, "recovery", i.cfg.Logger)
+	if err != nil {
+		return err
+	}
+
+	err = replaceConfTitle(targetConfPath, "recovery")
```
Comment (jimmykarily) [inline pkg/uki/upgrade.go:129]: Shouldn't we do this on installation too?

Comment (jimmykarily) [reply pkg/uki/upgrade.go:129]: Oh I see, you assume that from now on, the menus will [be correct from enki](https://github.com/kairos-io/enki/pull/77/files). Gotcha, all good.

**Review by jimmykarily — APPROVED** [review]

File: pkg/uki/common.go [hunk 2]
```diff
@@ -95,6 +100,55 @@ func replaceRoleInKey(path, key, oldRole, newRole string, logger sdkTypes.Kairos
 	return os.WriteFile(path, []byte(newContents), os.ModePerm)
 }
 
+func replaceTitle(role, title string) (string, error) {
+	var baseTitle string
+	passiveSuffix := " (fallback)"
```
Comment (Itxaka) [inline pkg/uki/common.go:105]: (nit): this should probably be in constants

**Review by Itxaka — APPROVED** [review]
noice

**Pull request merged**